examples/rich-text/webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+
+module.exports = {
+  entry: './client.js',
+  output: {
+    path: path.resolve(__dirname, 'static/dist'),
+    filename: 'bundle.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env']
+          }
+        }
+      }
+    ]
+  },
```

Comment:
As noted above, I don't think this is needed. Let's remove to keep as minimalist as possible.